packages/rx_bloc_cli/bin/compile.sh

```diff
@@ -9,5 +9,6 @@ dart run rx_bloc_cli create \
   --enable-feature-deeplinks=true\
   --enable-feature-widget-toolkit=true \
   --enable-social-logins=true \
+  --enable-internationalisation=true \
```

Comment:
Same as above

Reply:
This parameter was changed to `--enable-change-language`